pkg/v1/tkg/aws/client.go

```diff
@@ -295,9 +296,49 @@ func (c *client) GenerateBootstrapTemplate(i GenerateBootstrapTemplateInput) (*b
 	if !i.DisableTanzuMissionControlPermissions {
 		ensureTanzuMissionControlPermissions(&template)
 	}
+	ensureEBSCSiPermissions(&template)
 	return &template, nil
 }
 
+func ensureEBSCSiPermissions(t *bootstrap.Template) {
```

Comment:
I don't have context on these changes, gonna defer to @vuil

Reply:
This change is LGTM, needed because of an [upstream bug](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/3527).